docs/tutorials/first-custom-field.md

```diff
@@ -0,0 +1,78 @@
+# Creating Your First Custom Field
+
+A beginner-friendly guide to adding custom fields using Secure Custom Fields (SCF).
+
+**What is a custom field?**
+
+Custom fields let you add structured data to your content. With SCF, you can attach fields like text inputs, selects, image uploads, and more to your posts, pages, or custom post types.
+
+## Prerequisites
+
+-   SCF installed and activated
+-   A post type (default or custom) where you'll attach your fields
+-   Administrator access to WordPress
+
+You can learn more about WordPress basics here:
+
+-   [Theme Basics](https://developer.wordpress.org/themes/basics/)
+-   [Plugin Basics](https://developer.wordpress.org/plugins/plugin-basics/)
+
+## 1. Access the Admin Panel
+
+-   Go to **Secure Custom Fields → Field Groups** in your WordPress admin menu.
+-   Click **"Add New"** to create a new group of fields.
+
+## 2. Basic Configuration
+
+#### Field Group Title
+
+Give your field group a descriptive name, like `Movie Details` or `Product Specs`.
+
+#### Location Rules
+
+Choose where this group should appear. For example:
+
+-   Post type is equal to `Movie`
+-   Page template is `Product Page`
+
+This ensures your fields appear only where you need them.
```

Comment:
We could add a note saying that this fine-grained control allows the fields to be reused in different places.